src/questions/decentralized_exchanges.yaml

```diff
@@ -0,0 +1,786 @@
+  answerKeys:
+    - A
+  subTopics:
+    - crypto-exchange
+  difficultyLevel: Low
+  choices:
+    - content: Censorship resistance
+      key: A
+    - content: Fast matching
+      key: B
+    - content: Gas or Blockchain fees for orders
+      key: C
+    - content: No fees for cancelled orders
+      key: D
+- uuid: c55b4cf0-fcf9-4a33-8fae-4722ad825e1d
+  type: SingleChoice
+  content: Centralized exchanges do the market making by creating liquidity pools.
+  hint: Order Book Model
+  explanation: CEX follows Order Book Model whereas the DEX follows protocols(such as AMM) for creating the liquidity pools.
+  answerKeys:
+    - B
+  subTopics:
+    - amm
+  difficultyLevel: Low
+  choices:
+    - content: True
+      key: A
+    - content: False
+      key: B
+- uuid: 8d588484-6241-4f3f-b72c-1fc53bf11cd3
+  type: SingleChoice
+  content: ____ is a collection of funds, funded by Liquidity Providers(LP) that are locked in a smart contract, to facilitate trading, borrowing, lending, yielding, etc.
+  hint: NoHint
+  explanation: Liquidity Pools are created using a protocol(such as AMM) in DEX where liquidity providers come to fund the pools.
+  answerKeys:
+    - A
+  subTopics:
+    - amm
+  difficultyLevel: Low
+  choices:
+    - content: Liquidity Pools
+      key: A
+    - content: Protocol
+      key: B
+    - content: Order Book
+      key: C
+    - content: Exchange
+      key: D
+- uuid: 379a8445-faee-411c-a8bd-fe429268323a
+  type: SingleChoice
+  content: What does AMM stand for?
+  hint: NoHint
+  explanation: Automated Market Maker(AMM) is a protocol that creates the marketplace for buyers and sellers on its own.
+  answerKeys:
+    - B
+  subTopics:
+    - amm
+  difficultyLevel: Low
+  choices:
+    - content: Automated Market Mover
+      key: A
+    - content: Automated Market Maker
+      key: B
+    - content: Automated Maker Market
+      key: C
+    - content: None of the above
+      key: D
+- uuid: 70181494-66fe-41c7-b483-f6d6c0a66990
+  type: SingleChoice
+  content: What is an Automated Market Maker?
+  hint: Protocol, Liquidity and Automated
+  explanation: AMM is a type of DEX protocol that relies on a mathematical formula that allows digital assets to be traded without permission, automatically by using liquidity pools instead of the traditional order book pricing model.
+  answerKeys:
+    - D
+  subTopics:
+    - amm
+  difficultyLevel: Medium
+  choices:
+    - content: A protocol that automatically makes trades for a user
```

Comment:
We should reword this `A protocol that automatically makes trades for a user` as it can be a bit confusing